tools/shell/shell_runner.cpp

```diff
@@ -188,9 +188,9 @@ int main(int argc, char* argv[]) {
         auto shell = EmbeddedShell(database, conn, shellConfig);
         processRunCommands(shell, initFile);
         if (DBConfig::isDBPathInMemory(databasePath)) {
-            std::cout << "Opened the database under in-memory mode." << '\n';
+            std::cout << "Opening the database under in-memory mode." << '\n';
```

Comment:
I'm less sure about this change. Isn't the database already opened when users see this prompt? maybe @prrao87 can comment on this?